src/platform/forms-system/src/js/helpers.js

```diff
@@ -532,14 +536,15 @@ export function expandArrayPages(pageList, data) {
           generateArrayPages(arrayPages, data),
           nextPage,
         );
-        return _.assign(acc, {
+        return {
+          ...acc,
           lastArrayPath: null,
           arrayPages: [],
           currentList: newList,
-        });
+        };
       }
 
-      return _.set('currentList', currentList.concat(nextPage), acc);
+      return set('currentList', currentList.concat(nextPage), acc);
```

Comment:
I think it's safe to use spreading since `acc` is the accumulator param.

```suggestion
      return {
        ...acc,
        currentList: currentList.concat(nextPage),
      };
```